agent/bench-scripts/unittests

```diff
@@ -331,7 +326,6 @@ done
 
 # Final clean up, if there are no failures, we'll be able to remove the
 # hierarchy entirely.
-rm -rf $_testroot/opt $moongen_dir
```

Comment:
Should we keep the removal of `$_testroot/opt`?

Reply:
Added it back - got too hatchet-happy there...